web/src/pages/profile.tsx

```diff
@@ -293,6 +295,7 @@ const ProfilePage = (props: Props): ReactElement => {
     info: (
       <>
         <ProfileTabHeader tab="info" />
+        <ProfileErrorAlert fieldErrors={getFieldErrors()} />
```

Comment:
See the next line for an example on how we do it in other places

I copied and pasted this comment in three places

[Pebble] We had previously talked about how we should separate the UI logic from the conditional logic on whether it should display. I think we agreed it should be defined at the parent level. Therefore, fieldErrors does not need to be a prop.